src/types.ts

```diff
@@ -232,6 +232,12 @@ export interface CreateCredentialOptions {
    * If the issuer or holder does not list an `alg`, then the one specified in `header` will be used
    */
   header?: Partial<JWTHeader>
+
+  /**
+   * Allows including or overriding some header parameters for the resulting JWT.
+   */
+  jwt?: Partial<JWTOptions>
```

Comment:
I find this layering of options a little confusing.
Perhaps it would be easier to have:
```ts
export interface CreateCredentialOptions extends Partial<JWTOptions> {
```

And then call `createJWT` with `...options` instead of `...options.jwt`

Reply:
This would then also inject `options.removeOriginalFields` and `options.header` into the second argument of `createJWT`. Is this a problem?

Reply:
It's not a problem.
`createJWT` does not use those fields.